server/message_families/issuer_registration/setup.py

```diff
@@ -3,7 +3,7 @@
 import runpy
 from setuptools import setup, find_packages
 
-PACKAGE_NAME = "indy-catalyst-issuer-registration"
+PACKAGE_NAME = "aries-vcr-issuer-registration"
```

Comment:
@ianco , @nrempel, Is this going cause issues elsewhere?

Reply:
Don't think this will be an issue anymore